deploy/application/docker/tutorials/fluentd.md

```diff
@@ -23,6 +23,10 @@ Since you deploy microservices on Clever Cloud, you may need some data pipes bet
 
 Fluentd is an open source data collector written in Ruby, which lets you unify the data collection and consumption for a better use and understanding of data.
 
+{{< alert "info" "Why I can't just use Ruby runtime ?" >}}
```

Comment:
```suggestion
{{< alert "info" "Why can't I use the Ruby runtime ?" >}}
```

I had a hard time figuring out that the "ruby runtime" here is a ruby application. Maybe we could replace `the Ruby runtime` by `a Ruby application`? I don't know